test/llvm/lifetime-intrinsics/lifetime-no-return.chpl

```diff
@@ -0,0 +1,32 @@
+// This test is used to verify that llvm.lifetime instrinsics are being
+// emitted for procedures having no return statements (ret void)
+
+// Simple hack to retain elements to show up in LLVM IR with lifetime instrinsics
+proc refidentity(const ref a) const ref { return a; }
+
+proc mytest_no_return() {
+  const a : int = 32;
+  // CHECK: %[[REG1:[0-9]+]] = bitcast i64* %a_chpl to i8*
+  // CHECK-NEXT: call void @llvm.lifetime.start.p0i8(i64 8, i8* %[[REG1]])
+  refidentity(a);
+  var b : real = 3.99;
+  var c = b * 2 + a;
```

Comment:
apply refidentity to b and c so we don't have temporary variable names in the LLVM IR?